KappaAnalysis/src/Producers/MuonCorrectionsProducer.cc

```diff
@@ -181,7 +187,7 @@ void MuonCorrectionsProducer::Produce(KappaEvent const& event, KappaProduct& pro
 void MuonCorrectionsProducer::AdditionalCorrections(KMuon* muon, KappaEvent const& event,
                                    KappaProduct& product, KappaSettings const& settings) const
 {
-
+	// LOG(DEBUG) << "Additional Corrections applied! Please verify that this is correct ...\n";
```

Comment:
Why is this commented out?

Reply:
Because this function is always run this would also the debug output with unnecessary information. It is kept as a comment to remind one if it when implementing this function. It is currently empty.